client/performance-profiler/components/newsletter-banner.tsx

```diff
@@ -19,36 +22,52 @@ const Heading = styled.div`
 	font-weight: 500;
 	line-height: 24px;
 	text-align: left;
+	color: var( --studio-white );
 `;
 
 const Body = styled.div`
-	color: var( --studio-gray-70 );
+	color: var( --studio-gray-20 );
 `;
 
 const BlueberryButton = styled( Button )`
 	// && is needed for specificity
 	&& {
 		background: #3858e9;
 		border-color: #3858e9;
+
+		&:hover:not( :disabled ),
+		&:active:not( :disabled ),
+		&:focus:not( :disabled ) {
+			background-color: darken( #3858e9, 10% );
+			border-color: darken( #3858e9, 10% );
+			box-shadow: none;
+		}
 	}
 `;
 
 export const NewsletterBanner = ( { link }: { link: string } ) => {
 	const translate = useTranslate();
+	const isLoggedIn = useSelector( isUserLoggedIn );
 
 	return (
 		<Container>
 			<div>
-				<Heading>{ translate( "Sign up for weekly performance reports—it's free!" ) }</Heading>
+				<Heading>
+					{ translate( 'Get notified about changes to your site’s performance—it’s free!' ) }
+				</Heading>
 				<Body>
 					{ translate(
 						"Monitor your site's key performance metrics with a free report delivered to your inbox each week."
 					) }
 				</Body>
-				<Body>{ translate( 'All you need is a free WordPress.com account to get started.' ) }</Body>
+				{ ! isLoggedIn && (
+					<Body>
+						{ translate( 'All you need is a free WordPress.com account to get started.' ) }
+					</Body>
+				) }
 			</div>
 			<BlueberryButton variant="primary" href={ link }>
-				{ translate( 'Sign up for email reports' ) }
+				{ translate( 'Enable email alerts' ) }
```

Comment:
This should still read `{ translate( 'Sign up for email reports' ) }` for logged out users.